landing.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title> Landing Page</title>  
+    </head>
+    
+
+    <h1>Landing Page</h1>
+    <form>
+        <label for="username">username:</label><br>
+        <input type="text" id="user"><br>
+        <label for="password"> password:</label><br>
+        <input type="text" id="pass"><br>
+        <input type="button" value="Submit" onclick="redirect()"> 
+        
```

Comment:
Unnecessary empty line. Good formatting in your code is a lifesaver as projects grow in size, and especially once you start working with other people.